src/EnergyPlus/UnitarySystem.hh

```diff
@@ -880,6 +889,7 @@ namespace UnitarySystems {
     bool searchExhaustNodes(EnergyPlusData &state, const int nodeToFind, int &ZoneEquipConfigIndex, int &ExhaustNodeIndex);
     // void setSystemParams(EnergyPlusData &state, UnitarySys &thisSys, Real64 &TotalFloorAreaOnAirLoop, const std::string thisObjectName);
     bool searchTotalComponents(EnergyPlusData &state, std::string objectNameToFind, int &compIndex, int &branchIndex, int &airLoopIndex);
+    void setupAllOutputVars(EnergyPlusData &state, int const numAllSystemTypes);
```

Comment:
Adds a new function that wraps output variables for unitary system and coilsystemwater. This approach allows to setup the output variables without calling getinput multiple times.  The approach is adopted from CoilSystem-to-UnitarySystem branch.